src/reporters/udp_sender.js

```diff
@@ -19,34 +19,37 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+import * as constants from '../constants.js';
 import dgram from 'dgram';
 import fs from 'fs';
 import path from 'path';
 import {Thrift} from 'thriftrw';
 import Span from '../span.js';
 
 const HOST = 'localhost';
-const PORT =  5775;
+const PORT =  6832;
 const DEFAULT_UDP_SPAN_SERVER_HOST_PORT = `${HOST}:${PORT}`;
-const EMIT_SPAN_BATCH_OVERHEAD = 30;
 const UDP_PACKET_MAX_LENGTH = 65000;
 
 export default class UDPSender {
     _hostPort: string;
     _maxPacketSize: number;
+    _process: Process;
+    _emitSpanBatchOverhead: number;
     _maxSpanBytes: number;
     _spanBuffer: Array<Span>;
     _thriftBuffer: Buffer;
     _client: any;
     _spec: any;
     _byteBufferSize: number;
     _thrift: any;
+    _batch: Batch;
 
     constructor(hostPort: string = DEFAULT_UDP_SPAN_SERVER_HOST_PORT,
                 maxPacketSize: number = UDP_PACKET_MAX_LENGTH) {
         this._hostPort = hostPort;
         this._maxPacketSize = maxPacketSize;
-        this._maxSpanBytes = this._maxPacketSize - EMIT_SPAN_BATCH_OVERHEAD;
+        this._maxSpanBytes = this._maxPacketSize;
```

Comment:
it would be safer not to initialize it here (to incorrect vaue), but only do it in setProcess